test/integration/global_deploy_test.rb

```diff
@@ -77,7 +84,7 @@ def test_global_deploy_task_empty_selector_validation_failure
   end
 
   def test_global_deploy_task_success_selector
-    selector = "app=krane"
+    selector = "app=krane2"
```

Comment:
Why this change? Should we be using both this _and_ a generated selector?

Reply:
Before we always generated a selector, this PR allows us to pass the selector (needed for pruning reasons). But if the selector matches a different test's selector the pruning process conflicts.